client/js/widgets/widget.js

```diff
@@ -938,9 +943,14 @@ export class Widget extends StateManaged {
         let collection;
         let theItem;
         if(a.holder !== undefined) {
-          if(this.isValidID(a.holder,problems)) {
-            variables[a.variable] = widgets.get(a.holder).children().length;
+          const holders = asArray(a.holder).filter(w=> widgets.has(w) && widgets.get(w).get('type')=='holder');
+          if(holders.length != asArray(a.holder).length)
+            problems.push("Holder parameter must consist only of holders.");
```

Comment:
This changes the behavior of `COUNT`, right? It didn't enforce holders before so you could just count children of any widget.

Reply:
I fixed this (and simplified the code). I'm pretty confident that nothing else changes the behavior of any functions.